README.md

```diff
@@ -31,7 +161,9 @@ There's currenly only one BFF located in `api-dh-app` under `libs`. It is for `d
 
 Located under `.github/workflows`. There are:
 
-- `api-dh-app-ci.yml` - Used by the BFF for `dh-app`.
+- `api-dh-ci.yml` - Used by the BFF for `app-dh`.
 - `ett.yml` - Used by "Energy Track and Trace" app
 - `license-check-ci.yml` - Used for adding license to files
 - `workspace.yml` - Used to build, test, format and lint all front-end apps
+
+**Please note**: Before merging a frontend pull request, be sure that all checks have passed. Currently, jobs done by a Git-bot (like formatting and adding licenses) don't trigger a re-run of all the other checks. The "merge button" can therefore give you a false positive signal.
```

Comment:
question: Do you think we should add this to the Pull Request template as well? So the chance of seeing it is higher. Putting it here makes it a bit hidden.

Reply:
Good point, should be done now